Dockerfile

```diff
@@ -76,8 +76,9 @@ RUN echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontain
     && apt-get clean \
     && rm -rf /var/cache/apt /var/lib/apt/lists/*;
 
-# install umoci
-RUN curl -o /usr/local/bin/umoci -fsSL https://github.com/opencontainers/umoci/releases/download/v0.4.5/umoci.amd64 \
+# install umoci, retry with an exponential backoff strategy
+RUN curl --connect-timeout 5 --max-time 10 --retry 5 --retry-delay 0 --retry-max-time 40 -o /usr/local/bin/umoci \
```

Comment:
I will see if I can get `umoci` added to the `devel:kubic:libcontainers:stable` OBS project so we can eventually remove this and just install it like we do `skopeo`.